src/routes/DisciplineSelectionScreen.tsx

```diff
@@ -71,42 +71,34 @@ const DisciplineSelectionScreen = ({ route, navigation }: DisciplineSelectionScr
       return null
     }
     const selected = item.id === selectedId
-    const descriptionForWord = item.numberOfChildren === 1 ? labels.home.word : labels.home.words
-    const descriptionForUnit = item.numberOfChildren === 1 ? labels.home.unit : labels.home.units
-    const description = discipline.isLeaf ? descriptionForWord : descriptionForUnit
 
     return (
       <MenuItem selected={selected} item={item} onPress={() => handleNavigation(item)}>
         <ItemText>
           <BadgeLabel selected={selected}>{item.numberOfChildren}</BadgeLabel>
-          <Description selected={selected}>{description}</Description>
+          <Description selected={selected}>{childrenDescription(item)}</Description>
```

Comment:
I think this should be `childrenLabel` otherwise the number of words is displayed twic
![discipline_selection_num_words_twice](https://user-images.githubusercontent.com/28013368/144821244-bc841dbe-ea6c-4292-9086-b7ff1718f4d2.png)

Reply:
Good catch!